docs/getting-started.md

```diff
@@ -1,31 +1,22 @@
+In Arcs, a particle is a simple program that contains a bit of JavaScript and a [manifest](https://github.com/PolymerLabs/arcs/blob/master/runtime/manifest.md) file that describes what the particle does. A particle has a verb, can have inputs and outputs and may render UI to interact with the user. Arcs particles - potentially written by many different authors - can be combined and composed into Arcs recipes that perform a higher-level assistive function for the user. In our example the "Hello, World!" recipe combines together three particles to greet the user:
 
-
-
-*   The *HelloWorld* particle has no input and no UI. It simply outputs an entity of type "Message" that holds the greeting message: "Hello, World!".
-*   The *Greet* particle takes as input a singleton of type Message and renders it in the DOM.
+*   The *[HelloWorld](https://glitch.com/edit/#!/arcs-hello-world?path=particles/HelloWorld/HelloWorld.manifest)* particle has no input and no UI. It simply outputs an entity of type "Message" that holds the greeting message: "Hello, World!".
+*   The *[Greet](https://glitch.com/edit/#!/arcs-hello-world?path=particles/Greet/Greet.manifest)* particle takes as input a singleton of type Message and renders it in the DOM.
 
 These two particles would be enough to display a simple "Hello, World!" message. To demonstrate how particles can be used to compose UI, without needing to know about each other, we introduce a third particle:
 
-
-
-*   The *PersonalGreet *particle takes as input a singleton of type Person and displays a personal greeting message that gets composed into the DOM output from the previous Greet particle.
+*   The *[PersonalGreet](https://glitch.com/edit/#!/arcs-hello-world?path=particles/PersonalGreet/PersonalGreet.manifest)* particle takes as input a singleton of type Person and displays a personal greeting message that gets composed into the DOM output from the previous Greet particle.
 
 Every recipe runs inside an Arc. The Arc contains one or more recipes (the code and data flow) and views (the data) that get mapped into the Arc from the user's context (universe of all available data to the Arcs runtime).
 
-When the demo is loaded, the Arcs runtime does the following:
-
-
-
-*   Create a new Arc for the demo.
-*   Insert a Person entity named "Claire" into the Arc's context.
-*   Add the "Hello, World!" recipe to the Arc.
+When the demo is loaded, the Arcs runtime is initialized by including the [playground script](https://polymerlabs.github.io/arcs-cdn/v0.0.4/playground/playground.html). That script reads the Hello; World demo [arc.manifest](https://glitch.com/edit/#!/arcs-hello-world?path=arc.manifest) file and loads it as an Arc into the Arcs context (i.e., an Arc manifest is essecially a serialized Arc).
```

Comment:
Is the ';' in 'Hello; World' intentional?

Reply:
Also

s/essecially/essentially/

I wonder if the bit about 'an Arc manifest is essecially a serialized Arc' should be expanded (and possibly moved elsewhere if it ends up being too large). I think it could provide a window into how the system works.

Reply:
Woops. Fixed.